notebooks/Tutorial_templates/tutorial_template_msb_v1.ipynb

```diff
@@ -0,0 +1,139 @@
+{
```

Comment:
<p>I've seen Andy and other tutorial creators include comments next to each import indicating usage, which I find really helpful. This may be nice to add as guidance here. What might I put under <code>#other stuff </code>?  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nsidc/NSIDC-Data-Tutorials/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>